package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -30,11 +31,18 @@ function keys:write(value)
 	end
 end
 
+local min_password_length = ((site.config_mode or {}).remote_login or {}).min_password_length or 12
```

Comment:
You could reduce the code duplication a bit by using a variable for the remote login config:
```lua
local config = (site.config_mode or {}).remote_login or {}
```